basetemplates/generate-templates.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Render the latest bootkube templates and replace the old one in this directory.
+
+BOOTKUBE_VERSION=v0.9.1
+
```

Comment:
I would add the command to the bin folder

Here all command detect the root folder of project if required and switvch to this folder before executing their functionality. Here a relative file path is expected to exist. This seems to be such a case.

Reply:
I would keep the script in the basetemplate folder since this is only relevant for the rendered templates.

Apart from that, I fixed the base_dir handling in the generation script.